usaspending_api/api_docs/markdown/api_tutorial.md

```diff
@@ -25,13 +25,13 @@
 
 # Introductory Tutorial <a name="introduction"></a>
 
-Welcome to the introductory USASpending API tutorial. This tutorial is designed for people who aren't familiar with APIs and how to use them. If you already know what an "endpoint" is and the difference between GET and POST, you'll want to visit [Using the API](/docs/using-the-api), or jump right into the data with some [request recipes](/docs/recipes/).
+Welcome to the introductory USASpending API tutorial. This tutorial is designed for people who aren't familiar with APIs and how to use them. If you already know what an _endpoint_ is and the difference between `GET` and `POST`, you'll want to visit [Using the API](/docs/using-the-api) or jump right into the data with some [request recipes](/docs/recipes/).
 
 ## What's an API? <a name="whats-an-api"></a>
 
-"API" stands for "Application Programmer Interface." APIs make it easy for computer programs to request and recieve information in a format they can understand.
+_API_ stands for _Application Programmer Interface_. APIs make it easy for computer programs to request and receive information in a format they can understand.
```

Comment:
Ah, thank you for polishing us up... releasing with spelling errors would be embarrassing...